tensornetwork/backends/abstract_backend.py

```diff
@@ -706,6 +706,24 @@ def inv(self, matrix: Tensor) -> Tensor:
     raise NotImplementedError("Backend '{}' has not implemented `inv`.".format(
         self.name))
 
+  def pinv(self,
+           matrix: Tensor,
```

Comment:
change variable name to `tensor` (also in docstring)

Reply:
Changed, also in the docstring.